lib/ioki/apis/operator_api.rb

```diff
@@ -227,6 +227,15 @@ class OperatorApi
         base_path:   [API_BASE_PATH, 'products', :id],
         model_class: Ioki::Model::Operator::Area
       ),
+      Endpoints.custom_endpoints(
```

Comment:
This is how we have defined these endpoints in the platform API:

```ruby
      Endpoints::ShowSingular.new(
        :drt_area,
        base_path:   [API_BASE_PATH, 'products', :id],
        model_class: Ioki::Model::Platform::Area
      ),
      Endpoints::ShowSingular.new(
        :intermodal_area,
        base_path:   [API_BASE_PATH, 'products', :id],
        model_class: Ioki::Model::Platform::Area
      )
```

Which results in the methods:

- `drt_area(product_id)`
- `intermodal_area(product_id)`

In case you don't want to have the `areas_` prefix.

Reply:
👍 sounds good. I will change that :)